### ISSUE TITLE
Feature: return coordinates of selection box

### ISSUE THREAD

**Issue by kenzo23312** [valerybugakov/react-selectable-fast#97]
can we return coordinates of Selection box in onSelectionFinish, having many items causes app to lag, this might help with performance to look your selections by coordinates.

**Comment by heri16** [comment]
I agree that there is perceivable lag between mouseup and onSelectionFinished.

This project seems dead though. Maybe we could start a fork?
